webclient/src/components/AppSearchBar.vue

```diff
@@ -7,6 +7,14 @@ import type { SearchResponse } from "@/codegen";
 import { useFetch } from "@/utils/fetch";
 
 export default {
+  mounted() {
+    window.addEventListener("keydown", (e) => {
+      if (e.key === "/") {
+        e.preventDefault();
+        document.getElementById("search")?.focus();
+      }
+    })
```

Comment:
```suggestion
    window.addEventListener("keydown", (e) => {
      if (e.ctrlKey && e.key === "q")
        document.getElementById("search")?.focus();
    });
```

Reply:
https://stackoverflow.com/questions/11941180/available-keyboard-shortcuts-for-web-applications
Maybe `Ctrl + Shift + F` and `Ctrl + Shift + S`  would be a viable option then..